### DISCUSSION TITLE
Articles

### DISCUSSION THREAD

**Discussion by justinmilne-JH** [firemodels/fds#13380] [General]
This discussion is to request permission to reference NIST validation fire models for technical articles in the consulting industry. For the models, the fire models would be slightly adapted in order to provide greater visibility and credit to NIST and the FDS validation model. Is there a process for requesting formal permission to provide this content for technical articles?

**Comment by drjfloyd** [comment]
In general, no copyright attaches to US Government publications (i.e. they are in the public domain). No permission is needed to use the contents of the FDS documentation.  We would ask (but cannot require) that you provide attribution for anything that you use.

**Reply by justinmilne-JH** [reply]
Thank you for the prompt reply, Dr. Floyd. I will assure that all instances of using any NIST produced information are documented in this thread and will make sure to attribute credit to where it is due. Once again, thank you.
